3rdparty/osqp/CMakeLists.txt

```
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(osqp)
+
+include(ExternalProject)
+
+if("$ENV{ROS_DISTRO}" STRGREATER "kinetic")
```

@Naoki-Hiraoka even so, this change allows only build in `melodic` and `noetic`.

```suggestion
if("$ENV{ROS_DISTRO}" STRGREATER "indigo")
```

---

ありがとうございます。修正しました。